tools/setup.py

```diff
@@ -37,6 +36,10 @@
     'colorama>=0.3.7'
 ]
 
+_python_version = getattr(sys, "version_info", (0,))
```

Comment:
Why use `getattr` instead of call `sys.version_info` directly?

Reply:
@haroldrandom Fixed